src/inventory/db_utils.py

```diff
@@ -11,8 +11,8 @@
   "color_identity",
   "image_uris", # working
 
-  # commenting out to get DB up and running, FIXME still
-#   "card_faces", # needs work
+
+  "card_faces", # needs work
   # - currently stores as nan for single face cards, or
```

Comment:
I think the comments here are now old, and can be removed.

I believe you update `card_faces` to be an empty list instead of NaN

Reply:
This change is removing the comment.

Not sure what part of the code you are reading, but when I check the completed Dataframe there is no `card_faces` and the image uris have been captures for the front side.